packages/builder/src/components/design/settings/controls/ButtonConfiguration/ButtonConfiguration.svelte

```diff
@@ -0,0 +1,134 @@
+<script>
+  import DraggableList from "../DraggableList/DraggableList.svelte"
+  import ButtonSetting from "./ButtonSetting.svelte"
+  import { createEventDispatcher } from "svelte"
+  import { store } from "builderStore"
+  import { Helpers } from "@budibase/bbui"
+
+  export let componentBindings
+  export let bindings
+  export let value
+
+  const dispatch = createEventDispatcher()
+
+  let focusItem
+
+  $: buttonList = sanitizeValue(value) || []
```

Comment:
As discussed, removed the caching logic but keep the sanitization of the value to ensure that manifest default values get enriched as required.